app/globals.css

```
@@ -1,887 +1,464 @@
-    background-position: -200% 0;
-  }
+	0% {
+		background-position: 200% 0;
+	}
+
+	100% {
+		background-position: -200% 0;
+	}
 }
 
 @keyframes skeletonShimmer {
```

Stylelint (17.10.0)

[error] Expected keyframe name "skeletonShimmer" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)